src/qtgui/freqctrl.cpp

```diff
@@ -876,13 +878,11 @@ void CFreqCtrl::cursorEnd()
 
 void CFreqCtrl::setFrequencyFocus()
 {
-    uint8_t position = floor(log10(m_freq));
-    position = (uint8_t)fmax(position, 4);      // restrict min to 100s of kHz
+    if (!hasFocus() || m_ActiveEditDigit == -1) {
+        // Select last digit or 5th digit (100s of kHz), whatever is bigger.
+        uint8_t position = MAX(int(log10(m_freq)), 5);
```

Comment:
I'd suggest using `std::max` instead of a macro from `sys/param.h` (which is probably not available on all platforms, Windows in particular).

Also, `position` can be an `int`.

Reply:
Done, thx!